code/game/objects/items/devices/radio/headset.dm

```diff
@@ -349,6 +349,7 @@ GLOBAL_LIST_INIT(channel_tokens, list(
 			translate_binary = TRUE
 		if(keyslot2.syndie)
 			syndie = TRUE
+			canhear_range = 0
```

Comment:
Why is this here? Should this also be a define?

Reply:
eavesdropping syndi channel isn't good even if it's said unknown, so decreasing hearing range forcefully.
also, changed hearing range as a define and to -1. this will even prevent eavesdropping by standing below someone